python/cugraph-pyg/cugraph_pyg/data/cugraph_store.py

```diff
@@ -320,7 +320,13 @@ def __init__(
     def __del__(self):
         if self.__is_graph_owner:
             if isinstance(self.__graph._plc_graph, dict):
-                distributed.get_client().unpublish_dataset("cugraph_graph")
+                try:
+                    distributed.get_client().unpublish_dataset("cugraph_graph")
+                except Exception:
```

Comment:
I'm not familiar with the error, but is it possible to catch a more specific exception?  I'm just worried this has the potential to also catch typos and other things we should know about, and we forever end up just assuming it's not able to unpublish.
If catching everything is really better though for this, then feel free to ignore.